accelerator/apis.c

```diff
@@ -9,28 +9,35 @@
 #include "apis.h"
 #include "map/mode.h"
 
+static logger_id_t logger_id;
```

Comment:
You can define specific logger id in each file. For instance, `apis_logger_id`.

Reply:
Copy that!